modules/ROOT/partials/misc/url-handling.adoc

```diff
@@ -25,7 +25,7 @@ Example: path2/file.htm >> /path1/path2/file.htm
 
 === Domain absolute URLs
 
-This will convert all relative URLs to absolute URLs. The URLs will be absolute based on the <<document_base_url,document_base_url>> with domain.
+This will convert all relative URLs to absolute URLs. The URLs will be absolute based on the xref:document_base_url[document_base_url] with domain.
```

Comment:
```suggestion
This will convert all relative URLs to absolute URLs. The URLs will be absolute based on the xref:configure/url-handling.adoc#document_base_url[document_base_url] with domain.
```

Reply:
done